### ISSUE TITLE
[Bug]: Keyboard Focus indicator is not visible on selected link (like-Custom Properties) under MGTP’ page. 

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#24186]
### Describe the bug

Keyboard Focus indicator is not visible on selected link (like-Custom Properties) under MGTP’ page. 

**Actual Results:**
Keyboard focus indicator is not visible over the selected link (like-Custom Properties) under MGTP’ page.

**Expected Results:**
Keyboard focus indicators should be visible over the link (like-Custom Properties) under MGTP’ page.

### To Reproduce

**Steps to Reproduce:**

1. Open the[Components / mgt-agenda / Style - Custom Properties ⋅ Storybook](https://mgt.dev/next/v3/?path=/story/components-mgt-agenda-style--custom-properties) URL- 
2. The ‘Microsoft Graph Toolkit Playground' Overview page will get displayed. 
3. Navigate the 'Custom Properties’ link under components and select it. 
4. Renavigate to ‘Custom Properties’ link using arrow key. 
5. Observe the issue. 

**Attachments**

https://github.com/storybookjs/storybook/assets/93735775/f421d956-bec8-43db-90ff-88219aa0d0f3

![KB focus not visible (1)](https://github.com/storybookjs/storybook/assets/93735775/48667fb5-f9b9-4711-87e1-bf94633326c0)

### System

```bash
OS Build: Windows 11 
Version: 22H2 (OS Build 23443.1000) 
Browser: Edge dev 
Version 114.0.1793.0 (Official build) dev (64-bit)
```

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, As issue is still repro'ing, Any update here?

**Comment by Sidnioulz** [comment]
Hi @msftedad thanks for your patience 🙏 

I have a branch that completely rewrites the sidebar, with a proper `tree` role and many other improvements. On that branch, this issue is fixed. The branch is nearing completion and I'm hoping to see it land in the next minor release.

**Comment by msftedad** [comment]
@Sidnioulz, Thanks for the update and for all the work on improving the accessibility. Glad to hear that this issue is already fixed on the new branch with the proper tree role implementation. Looking forward to seeing these improvements in the upcoming release.
